src/components/Display.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] [MANDATORY] Kindly avoid the inline ESLint disabling rule, this is equally the same as altering or changing our linter files. Let us fix it

Reply:
Hehehehe this one took me a while to figure out why it wasn't working... lol..